.github/workflows/sanitizer-tests.yaml

```diff
@@ -101,10 +108,17 @@ jobs:
           git clone https://github.com/google/googletest.git --branch release-1.10.0
           cd ..
 
+      - name: Build Gtest (without -Wall)
```

Comment:
Reuse Fetch repositories already existing action and add argument to add or not discovery server